app/models/miq_ae_yaml_import.rb

```diff
@@ -15,8 +15,9 @@ def initialize(domain, options)
   def import
     if @options.key?('import_dir') && !File.directory?(@options['import_dir'])
       raise MiqAeException::DirectoryNotFound, "Directory [#{@options['import_dir']}] not found"
-    elsif !User.current_user.nil? && @options['zip_file'] && domain_locked?(@options['import_as'])
-      # raises exception only for a local import into the locked domain
+    elsif (!User.current_user.nil? && @options['zip_file'] && domain_locked?(@options['import_as'])) ||
```

Comment:
@pkomanek Have you tested this by trying to update or overwrite an existing git domain that was imported earlier. Would we allow the user to 
  - Import their Git Domain (which will get locked)
  - They push some changes into the Git Repo
  - They try to reimport (not update) into their Git Domain

Reply:
@mkanoor tested:
 * bin/update
 * git_import
 * git_reimport
 * git_update
 * zip_file_import

-> all works fine